lib/chef/knife/bootstrap.rb

```diff
@@ -67,6 +67,11 @@ class Bootstrap < Knife
         :description => "The ssh gateway",
         :proc => Proc.new { |key| Chef::Config[:knife][:ssh_gateway] = key }
 
+      option :ssh_gateway_identity,
+        :short => "-I SSH_GATEWAY_IDENTITY",
```

Comment:
This seems niche enough that it doesn't need a short option, might as well leave `-I` for something else in the future.